AirportTicketBooking.Tests/Services/UserServices/UserServiceTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using AirportTicketBooking.Interfaces.Repositories;
+using AirportTicketBooking.Models;
+using AirportTicketBooking.Services;
+using AirportTicketBooking.Tests.Services.UserService;
+using Moq;
+
+namespace AirportTicketBooking.Tests;
+
+public class UserServiceTests
+{
+    private readonly Mock<IUserRepository> _mockRepo;
+    private readonly UserService _service;
+
+    public UserServiceTests()
+    {
+        _mockRepo = new Mock<IUserRepository>();
+        _service = new UserService(_mockRepo.Object);
+    }
+
+    [Theory]
+    [ClassData(typeof(UserTestData))]
+    public async Task AddAsync_Should(User user)
+    {
+        _mockRepo.Setup(r => r.AddAsync(user)).ReturnsAsync(user.UserId);
+
+        var result = await _service.AddAsync(user);
+
+        Assert.Equal(user.UserId, result);
+        _mockRepo.Verify(r => r.AddAsync(user), Times.Once);
```

Comment:
Please add comments `// Arrange`, `// Act` and `// Assert` in this one and all other tests. Example:
```suggestion
        // Arrange
        _mockRepo.Setup(r => r.AddAsync(user)).ReturnsAsync(user.UserId);

        // Act
        var result = await _service.AddAsync(user);

        // Assert
        Assert.Equal(user.UserId, result);
        _mockRepo.Verify(r => r.AddAsync(user), Times.Once);
```